docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -40,6 +40,34 @@ At a minimum, ``hive.metastore.uri`` must be configured:
     connector.name=iceberg
     hive.metastore.uri=thrift://localhost:9083
 
+.. list-table:: **Iceberg configuration properties**
```

Comment:
Remove the bolding of the title .. if we want that as a general thing we do it across all tables or just update the CSS

Reply:
Understood! Done 👍🏽